### PR TITLE
Update scratch.md: include download binary tarball

### PR DESCRIPTION
**Pull request by xinsnake**
As `kubernetes-server-linux-amd64.tar.gz` is not provided with the tarball now, an extra step is required.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3933)

### CODE REVIEW

**Comment by xinsnake** [comment]
Just to comment to verify the CLA.

File: docs/getting-started-guides/scratch.md [hunk 1]
```diff
@@ -167,6 +167,8 @@ You can use a Kubernetes binary release (recommended) or build your Kubernetes b
 [Developer Documentation](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/).  Only using a binary release is covered in this guide.
 
 Download the [latest binary release](https://github.com/kubernetes/kubernetes/releases/latest) and unzip it.
+As server binary tarballs are no longer included in the Kubernetes final tarball. You need to locate and run
```
Comment (chenopis) [inline docs/getting-started-guides/scratch.md]: Suggested edit: "Server binary tarballs are no longer included in the Kubernetes final tarball, so you will need to locate and run.."

**Comment by chenopis** [comment]
@xinsnake ping

**Comment by xinsnake** [comment]
@chenopis  Sorry I thought no action required from me.  PR updated. Thanks!

**Pull request merged**